Base/res/html/misc/video-webm.html

```diff
@@ -79,8 +83,15 @@
             document.getElementById('selected').textContent = event.track.selected;
         };
 
+
+        video.addEventListener('timeupdate', () => {
+            const time = Number.parseFloat(video.currentTime).toFixed(2);
```

Comment:
`currentTime` and `duration` are already numbers, what does the `parseFloat()` do?